packages/core/core/src/PackagerRunner.js

```diff
@@ -94,3 +103,43 @@ export default class PackagerRunner {
     return contents;
   }
 }
+
+/*
+ * Build a mapping from async, url dependency ids to web-friendly relative paths
+ * to their bundles. These will be relative to the current bundle if `publicUrl`
+ * is not provided. If `publicUrl` is provided, the paths will be joined to it.
+ * 
+ * These are used to translate any placeholder dependency ids written during
+ * transformation back to a path that can be loaded in a browser (such as
+ * in a "raw" loader or any transformed dependencies referred to by url).
+ */
+function generateDepToBundlePath(
+  bundle: InternalBundle
+): Map<string, FilePath> {
+  let depToBundlePath: Map<string, FilePath> = new Map();
+  bundle.assetGraph.traverse(node => {
+    if (node.type !== 'dependency') {
+      return;
+    }
+
+    let dep = node.value;
+    if (!dep.isURL || !dep.isAsync) {
+      return;
+    }
+
+    let [bundleGroupNode] = bundle.assetGraph.getNodesConnectedFrom(node);
```

Comment:
Should BundleGraph have a `getEntryBundles` helper method? Seems relatively useful...

Reply:
👍

Reply:
thought right now PackagerRunner doesn't have access to the bundle graph. I might punt on this for now since it doesn't affect the public API here, but I agree it would be useful.